src/components/Layout/Layout.js

```diff
@@ -16,15 +18,64 @@ import Footer from '../Footer';
 injectTapEventPlugin();
 
 class Layout extends React.Component {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+    isLoggedIn: PropTypes.bool.isRequired,
+  };
+
+  state = {
+    drawerOpen: true,
+  };
+
+  handleDrawerToggle = () => {
+    this.setState({ drawerOpen: !this.state.drawerOpen });
+  };
 
   render() {
+    if (!this.props.isLoggedIn) {
+      return (
+        <MuiThemeProvider muiTheme={this.context.muiTheme}>
+        <div>
+          <Header
+          />
+          {this.props.children}
+          {// <Feedback />
+          }
+          <Footer
+          isLoggedIn={this.props.isLoggedIn}
+          />
+        </div>
+        </MuiThemeProvider>
+      );
+    }
+    const contentStyle = {  transition: 'margin-left 450ms cubic-bezier(0.23, 1, 0.32, 1)' };
+
+    if (this.state.drawerOpen) {
+      contentStyle.marginLeft = 256;
+    }
+
     return (
       <MuiThemeProvider muiTheme={this.context.muiTheme}>
-        <div>
-          <Header />
+      <div>
+        <div style={contentStyle}>
+          <Navigation 
```

Comment:
After some discussion, DrawerItems is spatially linked to Layout, so we can leave it there. But the Navigation component can be added to Header, so that any data flow can be directed to the Header without having to consider any stray Navigation components like in Layout.js. Also, a lot of redundant code will be removed because of this and conditional rendering can only be used for DrawerItems.